src/main/java/com/example/model/Book.java

```diff
@@ -0,0 +1,34 @@
+package com.example.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+@Entity
```

Comment:
Don't use Lombok's Data on JPA entities, it bundles several annotations, some of which (ToString, EqualsAndHashcode) can be problematic for persisted entities. Use @ Getter Setter for now